docs/openapi/api-private-v1.yaml

```diff
@@ -125,6 +172,57 @@ components:
           type: array
           items:
             $ref: './schemas-pn-timeline-v1.yaml#/components/schemas/TimelineElement'
+    
+
+    ############################################################################################
+    ###                 COMUNICAZIONE DELLA VISUALIZZAZIONE DI UNA NOTIFICA                  ###
+    ############################################################################################
+
+    RequestNotificationViewedDto:
+      type: object
+      required:
+        - iun
+        - recipientType
+        - recipientInternalId
+        - raddType
+        - raddBusinessTransactionId
+        - raddBusinessTransactionClosingDate
+      properties:
+        iun:
+          type: string
+          description: Identificativo Univoco della Notifica
+        recipientType:
```

Comment:
Mettere a fattor comune con quello in api-in/external-b2b-webook

Reply:
Messa la definizione in un file separato e usato $ref in entrambi i file